### PR TITLE
docs: Azure docs update

### PR DESCRIPTION
**Pull request by vsmalladi**
### Description

Updated azure tutorial and added in Azure batch into cloud

### QC

* [ ] The PR contains a test case for the changes or the changes are already covered by an existing test case.
* [x] The documentation (`docs/`) is updated to reflect the changes or this is not necessary (e.g. if the change does neither modify the language nor the behavior or functionalities of Snakemake).

### CODE REVIEW

**Comment by vsmalladi** [comment]
@johanneskoester updated docs and ran through them. This should reflect all the commands in the tutorial to get the keys.

**Comment by vsmalladi** [comment]
@jakevc  was already working on this while you too were working on it. Should be able to just use this one and merge in changes.

**Comment by jakevc** [comment]
shoot sorry yeah I guess we are on the same schedule haha

File: docs/executing/cloud.rst [hunk 1]
```diff
@@ -454,3 +454,106 @@ the `FUNNEL_SERVER_USER` and  `FUNNEL_SERVER_PASSWORD` AS environmental variable
...
+
+Using this executor typically requires you to start with large data files
+already in Azure Storage, and then interact with them via Azure Batch. An easy way to do this is to use the
+`azcopy <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10>`__.
+command line client. For example, here is how we might upload a file
+to storage using it:
+
+.. code-block:: console
+
+    $ azcopy copy mydata.txt "az://snakemake-bucket/1/mydata.txt"
```
Comment (jakevc) [inline docs/executing/cloud.rst:480]: how does this "az://" protocol work ?  I have not ever seen that nor can I find evidence of it in the azcopy docs ?

Comment (vsmalladi) [reply docs/executing/cloud.rst:480]: sorry this should be the other way

azcopy copy mydata.txt "https://$account.blob.core.windows.net/snakemake-bucket/1/mydata.txt"

Comment (vsmalladi) [reply docs/executing/cloud.rst]: No i wish. I will try to work on it for azcopy.

**Comment by johanneskoester** [comment]
Am I right that this can be closed now as @jakevc has merged it into his PR?

**Comment by vsmalladi** [comment]
Yep this can close

**Pull request closed without merging**